src/plugins/action_create_user/src/createUser.ts

```diff
@@ -1,4 +1,4 @@
-module.exports['createUser'] = async function (user: any, DBConnect: any) {
+module.exports['createUser'] = async function ({ aplicationData, ...user }: any, DBConnect: any) {
```

Comment:
Spelling: applicationData :)

Reply:
oops, i should really test this (copied from demo branch but changed slightly).

I tested for user rego, seems to be working fine. cheers !

Reply:
2bdd5a77e62ec0e2cb7af272f902dec2977c83e7